File: app/src/main/java/com/example/testlunu/net/acu/cityinfo/SupplementalAdminArea.java
package com.example.testlunu.net.acu.cityinfo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class SupplementalAdminArea {

    @SerializedName("Level")
    @Expose
    public Integer level;
    @SerializedName("LocalizedName")
    @Expose
    public String localizedName;
    @SerializedName("EnglishName")
    @Expose
    public String englishName;

}

File: app/src/main/java/com/example/testlunu/net/acu/cityinfo/SupplementalAdminArea.kt
package com.example.testlunu.net.acu.cityinfo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class SupplementalAdminArea (
    @SerializedName("Level")
    @Expose
    var level: Int?,
    @SerializedName("LocalizedName")
    @Expose
    var localizedName: String?,
    @SerializedName("EnglishName")
    @Expose
    var englishName: String?
)